src/components/Breadcrumb/__tests__/__snapshots__/Breadcrumb-test.js.snap

```diff
@@ -53,7 +53,7 @@ exports[`Breadcrumb should render <Breadcrumb /> with a single item 1`] = `
         className="separator"
         dangerouslySetInnerHTML={
           Object {
-            "__html": " > ",
+            "__html": ">",
```

Comment:
I actually think that this is not correct, since there won't be spaces around the separator anymore (unless they are manually put?)?

What happens if you copy paste now, vs previous?

Reply:
See https://github.com/algolia/instantsearch.js/pull/3176#discussion_r223459777

Reply:
Do you mean using [Enzyme's `html()`](https://airbnb.io/enzyme/docs/api/ShallowWrapper/html.html)?

It would output like this:

```html
"<div class="root noRefinement"><ul class="list"><li class="item"><a class="link">home</a></li><li class="item selectedItem"><span class="separator" aria-hidden="true">&gt;</span><!-- react-text: 7 -->name0<!-- /react-text --></li></ul></div>"
```